### PR TITLE
Added golang and node process utils

### PR DESCRIPTION
**Pull request by grariel**
Most of the code is copied from:
https://github.com/Granulate/gprofiler/blob/master/gprofiler/profilers/perf.py#L272

And will be used there as well.

### CODE REVIEW

**Comment by michelhe** [comment]
put these in a their own module, don't throw it in the package root

**Comment by grariel** [comment]
> put these in a their own module, don't throw it in the package root

Python and Java are also in the root folder.

File: granulate_utils/node.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+
+DETECTED_NODE_PROCESSES_REGEX = r"^.+/libnode\.so"
```
Comment (Jongy) [inline granulate_utils/node.py:6]: There is no such thing as `libnode`. Node process identification is done by `/proc/pid/exe`.

Comment (grariel) [reply granulate_utils/node.py]: Apparently it's only loaded on some versions. Changed to gProfiler's method, so now all the code is copied.

**Comment by michelhe** [comment]
See that the runtime detection code in gProfiler has gone through some changes here :
https://github.com/Granulate/gprofiler/pull/460

File: granulate_utils/golang.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+from granulate_utils.linux.elf import read_elf_symbol, read_elf_va
+
+
+def is_golang_process(process: Process) -> bool:
+    return get_process_golang_version(process) is not None
+
+
+@functools.lru_cache(maxsize=4096)
+def get_process_golang_version(process: Process) -> Optional[str]:
+    elf_path = f"/proc/{process.pid}/exe"
```
Comment (michelhe) [inline granulate_utils/golang.py:21]: use `granulate_utils.linux.process.process_exe`

**Review by michelhe — APPROVED** [review]

**Pull request merged**